test/sinon.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+
+const s = document.createElement('script');
+
+// on IE11 and Safari 9, load the last supported sinon version
+if (/(?:MSIE|Trident\/7.0|Version\/9.*Safari)/.test(navigator.userAgent)) {
+  s.src = 'https://unpkg.com/sinon@9.2.4/pkg/sinon-no-sourcemaps.js';
+} else {
+  s.src = '/test/base/node_modules/sinon/pkg/sinon.js';
+}
+
+document.write(s.outerHTML);
```

Comment:
evil stuff until we get rid of IE11

Reply:
should we only do this in the ie 11/safari 9 if statement?

Reply:
no, we have to write out the sinon URL in all cases. Once we drop IE11/Safari 9, we can remove this file and only include sinon in karma, since right now, the current version is only served and not included by default (https://github.com/videojs/video.js/pull/7299/files#diff-8cbe9d03b9fa5e04ef3fa3f8be9763779e57a2b026d5c70cb2ed7afcef0d6df5R43). Trying to limit scripts erroring out in IE11.